bin/index.js

```diff
@@ -21,7 +18,9 @@ server.listen(config.port, err => {
     return;
   }
 
-  console.log(`micro-airtable-api listening on port ${config.port}`);
+  console.log(
+    `> micro-airtable-api listening on http://localhost:${config.port}`
```

Comment:
Unrelated to the larger goals of the PR, but I added the `http://localhost:` bit here, since that makes the URL a clickable link from some terminals (eg. iTerm, Hyper).